go/vt/vtgateproxy/discovery.go

```diff
@@ -31,14 +30,8 @@ import (
 	"vitess.io/vitess/go/vt/log"
 )
 
-var (
-	jsonDiscoveryConfig = flag.String("json_config", "", "json file describing the host list to use fot vitess://vtgate resolution")
```

Comment:
Seemed clearer to put all the flag definitions in the same place in the main vtgateproxy file.